data_access.go

```diff
@@ -169,6 +205,15 @@ func (s *DataAccessSyncer) importDataAccess(config *data_access.DataAccessSyncCo
 	}
 }
 
+func isSnowFlakeDefaultRole(s string) bool {
+	for _, u := range SNOWFLAKE_DEFAULT_ROLES {
```

Comment:
No other way to detect this (from a column from the SHOW ROLES)?

Reply:
In show roles you have three fields (is_inherited, is_default and is_current). However in the snowflake docs they are not defined and looking at their values in our instance we would not get roles like "OrgAdmin, Masking_Admin" in our list. If I would do a select on show roles where is_inherited OR is_current the following roles match 
OKTA_PROVISIONER
PUBLIC
SECURITYADMIN
SYSADMIN
USERADMIN
If this is enough then we go for this option

Reply:
Weird. Ok to keep it hardcoded then
But the list should probably be this: https://docs.snowflake.com/en/user-guide/security-access-control-overview.html#system-defined-roles

MASKING_ADMIN (which you have in the list now), for example, is something we made to test, I think.

Reply:
ah but if MASKIND_ADMIN is ours then I can probably switch to the dynamic list. I am not a huge fan of hardcoding either so I rather go for dynamic then

Reply:
seems is_default just shows the default role so that is of no use. is_inherited does not cover AccountAdmin etc so I suppose we should go with the hardcoded list based of the URL you provided, i'll udpate the PR